examples/utils/migration-tools/src/simpleLoader/sessionStorageSimpleLoader.ts

```diff
@@ -0,0 +1,64 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import {
+	type ICodeDetailsLoader,
+	type IContainer,
+} from "@fluidframework/container-definitions/internal";
+import type { ITelemetryBaseLogger } from "@fluidframework/core-interfaces";
+import {
+	createLocalResolverCreateNewRequest,
+	LocalDocumentServiceFactory,
+	LocalResolver,
+	LocalSessionStorageDbFactory,
+} from "@fluidframework/local-driver/internal";
+import { LocalDeltaConnectionServer } from "@fluidframework/server-local-server";
+import { v4 as uuid } from "uuid";
+
+import type { ISimpleLoader } from "./interfaces.js";
+import { SimpleLoader } from "./simpleLoader.js";
+
+const urlResolver = new LocalResolver();
+
+const localServer = LocalDeltaConnectionServer.create(new LocalSessionStorageDbFactory());
+
+/**
+ * @alpha
+ */
+export class SessionStorageSimpleLoader implements ISimpleLoader {
+	public constructor(
+		private readonly codeLoader: ICodeDetailsLoader,
+		private readonly logger?: ITelemetryBaseLogger,
+	) {}
+
+	public async supportsVersion(version: string): Promise<boolean> {
+		return true;
+	}
+
+	public async createDetached(
+		version: string,
+	): Promise<{ container: IContainer; attach: () => Promise<string> }> {
+		const documentId = uuid();
+		const loader = new SimpleLoader({
+			urlResolver,
+			documentServiceFactory: new LocalDocumentServiceFactory(localServer),
+			codeLoader: this.codeLoader,
+			logger: this.logger,
+			generateCreateNewRequest: () => createLocalResolverCreateNewRequest(documentId),
+		});
+		return loader.createDetached(version);
+	}
+	public async loadExisting(id: string): Promise<IContainer> {
+		const documentId = id;
+		const loader = new SimpleLoader({
+			urlResolver,
+			documentServiceFactory: new LocalDocumentServiceFactory(localServer),
+			codeLoader: this.codeLoader,
+			logger: this.logger,
+			generateCreateNewRequest: () => createLocalResolverCreateNewRequest(documentId),
+		});
+		return loader.loadExisting(`${window.location.origin}/${id}`);
```

Comment:
We almost don't need `SessionStorageSimpleLoader` (and can instead just use SimpleLoader directly with the local server arguments), but unfortunately `LocalResolver` expects a full URL here (despite not actually caring about the host - a typical flaw of the url resolver pattern).  Whereas `InsecureTinyliciousUrlResolver` can handle a direct "id" request.

I might see if we could make/use a less-opinionated url resolver here as a followup, just so we don't have to have this deviation.